# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libv4l/no-gconv.patch.cross
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- v4l-utils-1.26.0/contrib/meson.build.vanilla	2023-11-26 16:55:45.630890662 +0100
+++ v4l-utils-1.26.0/contrib/meson.build	2023-11-26 16:56:03.372889632 +0100
@@ -9,5 +9,5 @@
 subdir('decode_tm6000')
 
 if have_gconv
-    subdir('gconv', if_found : [dep_jis, dep_jisx0213])
+  # subdir('gconv', if_found : [dep_jis, dep_jisx0213])
 endif
